src/integrationtests/java/com/aws/greengrass/integrationtests/NucleusLaunchUtils.java

```diff
@@ -127,6 +129,14 @@ void startNucleusWithConfig(NucleusLaunchUtilsConfig config) throws InterruptedE
                     .maxRetryInterval(Duration.ofSeconds(1))
                     .retryableExceptions(Collections.singletonList(RetryableException.class))
                     .build();
+            Retryer retryer =  (retryConfig1, request, context) ->
+            RetryUtils.runWithRetry(retryConfig,
+                () -> {
+                    request.execute(context);
+                    return null;
+                },
+                "test-setup", LogManager.getLogger(getClass()));
+            SyncHandler.setRetryer(retryer);
```

Comment:
Without this, the `RetryConfig` of the sync strategy classes is overriden to default as new strategy class is created based on the config during the shadow manager install (`kernel.launch()`)